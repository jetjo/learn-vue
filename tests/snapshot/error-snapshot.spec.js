import { expect, it } from "vitest";

it('to learn how to expect error throwing', () => {
	expect(() => {
		throw new Error('error')
	}).toThrowErrorMatchingSnapshot('用于测试的提示信息, 并不是期待的异常的message');
})