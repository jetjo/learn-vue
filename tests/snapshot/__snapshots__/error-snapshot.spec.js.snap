// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`to learn how to expect error throwing > 提示 1`] = `"error"`;

exports[`to learn how to expect error throwing > 用于测试的提示信息, 并不是期待的异常的message 1`] = `"error"`;
